src/common/NavigationBar/styled.js

```diff
@@ -111,7 +111,7 @@ export const StyledNavigationLink = styled(NavLink)`
     padding: 8px 24px;
     align-items: center;
     gap: 10px;
-    border: 1px solid ${({ theme }) => theme.color.white};
+    border: ${({ noBorder }) => (noBorder ? "none" : "1px solid")};
```

Comment:
Perhaps keeping the current border and adding the following below would be a good approach:

    ${({ $pageButton }) => $pageButton && css`
        border: none;
    `}

Reply:
Ok, done. Now it's ok? :)